presto-docs/src/main/sphinx/release/release-0.189.rst

```diff
@@ -48,8 +48,6 @@ JDBC Driver Changes
   running arbitrary SQL on behalf of users. For programmatic use, continuing
   to use ``setCatalog()`` and ``setSchema()`` on ``Connection`` is recommended.
 * Allow executing ``SET SESSION`` and ``RESET SESSION``.
-* Change mapping for columns with ``DECIMAL(p,s)`` data type from Presto ``DOUBLE``
-  type to the corresponding Presto ``DECIMAL`` type.
```

Comment:
as described by @losipiuk in https://github.com/prestodb/presto/issues/9327#issuecomment-343694488, this aren't changes in the Presto's JDBC client lib, but they are in the presto jdbc connectors (`presto-base-jdbc`, so mysql, postgresql, sqlserver connectors)